src/CSET/recipes/generic_level_domain_mean_time_series_case_aggregation_hour_of_day.yaml

```
@@ -1,5 +1,5 @@
 category: $LEVELTYPE Level Time Series
-title: $VARNAME $LEVEL domain mean time series aggregation by hour of day
+title: "$VARNAME $LEVEL domain mean time series \n Aggregation by hour of day"
```

```suggestion
title: "$VARNAME $LEVEL domain mean time series\n Aggregation by hour of day"
```
Don't have a space before a newline. It isn't needed and can cause the layout to be lop-sided.